packages/hooks/src/test/index.test.js

```diff
@@ -943,3 +944,17 @@ test( 'checking hasFilter with named callbacks and removeAllActions', () => {
 	expect( hasFilter( 'test.filter', 'my_callback' ) ).toBe( false );
 	expect( hasFilter( 'test.filter', 'my_second_callback' ) ).toBe( false );
 } );
+
+test( 'async filter', async () => {
```

Comment:
This looks great. :) 

Can we have an example with actions?

Reply:
Yes, if you like it I'll finish the PR: add a full test suite and documentation 🙂

Reply:
I added a few more tests. I'd also like to add more documentation, but the `hooks` package is not very well documented in the first place. It's like if it was assumed that everyone already knows how WordPress actions and filters work. There are `*.md` guides documenting specific actions and filters, but the API itself is not documented. Except some jsdoc comments which are however not propagated to any `*.md` docs.